NewHorizons/Handlers/EyeDetailCacher.cs

```diff
@@ -0,0 +1,39 @@
+using NewHorizons.Utility;
+using NewHorizons.Utility.OWML;
+using System.Linq;
+
+namespace NewHorizons.Handlers;
+
+public static class EyeDetailCacher
+{
+    public static void Init()
+    {
+        foreach (var body in Main.BodyDict["EyeOfTheUniverse"])
+        {
+            NHLogger.LogVerbose($"{nameof(EyeDetailCacher)}: {body.Config.name}");
+            if (body.Config?.Props?.details == null) continue;
+
+            foreach (var detail in body.Config.Props.details)
+            {
+                NHLogger.LogVerbose($"{nameof(EyeDetailCacher)}: {detail.path}");
+
+
+                if (string.IsNullOrEmpty(detail.path)) continue;
+
+                var planet = detail.path.Contains('/') ? detail.path.Split('/').First() : string.Empty;
+
+                // TODO: what other root paths can we ignore
```

Comment:
I don't think there is any other paths since everything eye related is under these 2